plugin/pkg/admission/resourcequota/apis/resourcequota/types.go

```diff
@@ -68,5 +68,5 @@ type LimitedResource struct {
 	// "PriorityClassNameIn=cluster-services"
 	// +optional
 	//	MatchScopes []string `json:"matchScopes,omitempty"`
-	MatchScopes []core.ScopedResourceSelectorRequirement `json:"matchScopes,omitempty"`
+	MatchScopes []corev1.ScopedResourceSelectorRequirement `json:"matchScopes,omitempty"`
```

Comment:
I didn't expect this.  One the other hand, I'm hard pressed to say why it shouldn't be allowed.

@liggitt we have anything that speaks against this?

Reply:
@deads2k @liggitt if you notice, in `v1alpha1` and `v1beta1` this field is external but this one isn't. looks really odd to me